corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -1,9 +1,26 @@
+hqImport("hqwebapp/js/initial_page_data").get('user_types')
 hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix'], function ($, kissAnalytics) {
     $(function () {
         if (document.location.href.match(/reports\/case_list/)) {
             $(document).on('click', 'td.case-name-link', function () {
                 kissAnalytics.track.event("Clicked Case Name in Case List Report");
             });
+            var user_types = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            $(document).on('click', 'button#apply-filters', function () {
+                kissAnalytics.track.event("Clicked Apply",
+                    {"filters": _.map(_.find($('#paramSelectorForm').serializeArray(),
+                            function(obj)
+                            {
+                                return obj.name==="case_list_filter"
+                            }).value.split(','),
+                            function(item)
+                            {
+                                if(item[0]==="t")
+                                {
+                                    return user_types[item.substring(3)]
+                                } else {
+                                    return item}})});
```

Comment:
Could you clean up this indentation and bracketing? looks like stickler had an error

Reply:
https://github.com/dimagi/commcare-hq/pull/22079/commits/3401d397d18af1ff745b72f2f953303068250aca